controllers/v1/solutions.js

```diff
@@ -4,11 +4,133 @@
  * created-date : 03-Sep-2020
  * Description : Solution related information.
  */
-
+// Dependencies
+const solutionsHelper = require(MODULES_BASE_PATH + "/solutions/helper");
 module.exports = class Solutions extends Abstract {
     
     constructor() {
         super(schemas["solutions"]);
     }
 
+      /**
```

Comment:
@aman-tunerlabs - As discussed we will remove this API from here, create one improvement project solution create an API in unnati that sets all the default value as required.